doc/sphinx/source/community/backward_compatibility.rst

```diff
@@ -0,0 +1,263 @@
+this work. This requires active involvement from both the people
+developing the new features and the *recipe developers*. ESMValTool core
+developers and ESMValCore core developers need to make sure they clearly
+communicate changes. In the first place, this is done by writing good
+descriptions in issues and pull requests on GitHub, but some of this
+material also makes it to the changelog (where the GitHub pull requests
+are linked). It is highly recommended to communicate a relevant
+selection (e.g. important new, scheduled for removal, and removed
+features) also by other means, to ensure we reach as many people
+potentially affected as possible (see “Guidance on handling
+*backward-incompatible changes*” section below).
+We organize monthly community meetings where *recipe developers* can learn about
+the latest developments and everyone is welcome to join, ask questions, and
+provide feedback.
+
+To meet the needs of users and funders, we should take reproducibility
+of older results seriously, but this should not hold us back from
+advancing our tools. We can support this by uploading a well tested
+container image to an archive that provides a DOI and by providing clear
+instructions on how to use such containers.
+
+Helping developers to upgrade
+-----------------------------
+
+*Recipe users* of ESMValTool will be able to successfully run
+*integrated recipes* using a *release*, since all
+*backwards-incompatible changes* introduced between *releases* will have
+been fixed before the *release* is created. Please note the
+:ref:`broken recipe policy<broken-recipe-policy>`.
+
+However, *recipe developers* working on *user recipes* must be provided
+with information to enable them to adapt their code to resolve issues
+related to *backwards-incompatible changes* when *backwards-incompatible
+changes* are introduced to the main branch / when a *release* of
+ESMValTool is created.
+
+Guidance on handling *backward-incompatible changes*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As well as helping users to handle *backward-incompatible changes*, the
+policy and surrounding tools must help developers avoid making
+*backwards-incompatible changes*. Not many ideas developed on this yet,
+but components should include:
+
+-  Testing; *backwards-incompatible changes* should be discovered as
+   early in the development process as possible. This motivates
+   continued investment in automated testing.
+-  Guidance on how to minimise the likelihood of introducing
+   *backward-incompatible changes* and how to use deprecation warnings
+   when needed (see `developer
+   guidance <https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#backward-compatibility>`__).
+-  Instructions on how to provide the information that *recipe
+   developers* need to adapt to the *backward-incompatible change*,
+   e.g. the text that will go into *release* notes if we follow the
+   proposal above.
```

Comment:
This needs to be clarified/resolved before merging this. Once merged, this isn't a proposal anymore but the actual policy.

Reply:
It was indeed an unclear point. Is this better: https://github.com/ESMValGroup/ESMValTool/pull/2879/commits/5bc685b158f9cecbe5e9c5b04016e2297b1cd8dd ?

Reply:
Much better, thanks!